Tests/CartonCommandTests/CommandTestHelper.swift

```diff
@@ -246,6 +246,7 @@ public extension XCTest {
     command: String,
     cwd: URL? = nil, // To allow for testing of file based output
     expected: String? = nil,
+    expectedContains: Bool = false,
```

Comment:
Can a doc comment be added for this function? It's not quite clear what `expectedContains` is supposed to indicate, and a doc comment could clarify that and also increase our documentation coverge in general.

Reply:
Comments added